core/src/main/java/org/frankframework/http/rest/ApiListener.java

```diff
@@ -64,7 +66,7 @@ public class ApiListener extends PushingListenerAdapter implements HasPhysicalDe
 	private @Getter boolean updateEtag = AppConstants.getInstance().getBoolean("api.etag.enabled", false);
 	private @Getter String operationId;
 
-	private @Getter HttpMethod method = HttpMethod.GET;
+	private List<HttpMethod> method = Arrays.asList(HttpMethod.GET);
```

Comment:
I believe that in Java11 we can now use `List.of(...)`?

Reply:
Correct, I'll replace this for all occurences of `Arrays.asList()` in the `org.frankframework.http.rest` package